.github/workflows/linuxPullRequest.yml

```diff
@@ -2,6 +2,11 @@ name: CI Linux PR
 on:
   pull_request:
     branches: [ main ]
+  workflow_dispatch:
```

Comment:
I would not add this here because when you trigger this workflow by hand github.event.pull_request.head.ref is not set so the workflow will not work.